Configuration.props

```diff
@@ -37,10 +51,16 @@
     <AllSupportedTargetAndroidAbis>$(AllSupported32BitTargetAndroidAbis);$(AllSupported64BitTargetAndroidAbis)</AllSupportedTargetAndroidAbis>
   </PropertyGroup>
   <PropertyGroup>
+    <_MingwPrefixTail Condition=" '$(HostOS)' == 'Darwin' ">.static</_MingwPrefixTail>
+    <_MingwCommandPrefix32>i686-w64-mingw32$(_MingwPrefixTail)</_MingwCommandPrefix32>
```

Comment:
I should have been more explicit, but I was suggesting that this be exported as `$(MingwCommandPrefix32)` and `$(MingwCommandPrefix64)` properties. The lack of a `Condition` means that these shouldn't be overridden.